buildSrc/build.gradle

```diff
@@ -15,6 +15,6 @@ repositories {
 
 dependencies {
     implementation  group: 'gradle.plugin.org.jetbrains.intellij.plugins', name: 'gradle-intellij-plugin', version: '0.4.14'
-    testImplementation 'org.spockframework:spock-core:2.0-groovy-3.0'
+    testImplementation 'org.spockframework:spock-core:2.3-groovy-4.0'
```

Comment:
@sethdarragile6 Let's delete this line since there are no tests in this project.